arkouda/pdarraycreation.py

```diff
@@ -49,6 +49,21 @@
 ]
 
 
+def _infer_from_size(size):
+    shape: Union[int_scalars, Tuple[int_scalars, ...]] = 1
+    if isinstance(size, tuple):
```

Comment:
this seems to do the same thing as `_calc_shape` that I added to `util` a while back, we can probably combine them 
https://github.com/Bears-R-Us/arkouda/blob/844ad2fd73cd49a93ed5057f2ef5f5e7df061d8a/arkouda/util.py#L613-L615

I actually like the name of urs a lot though. Maybe `_infer_shape_from_size`?